# Current command sent to roboclaw with timestamp - plane

Header header
float64 m1
float64 m2
float64 m3
float64 m4
float64[] xdes
float64[] ydes
uint8 numrobot
float64 bx
float64 by
float64 fx
float64 fy
